argilla-server/src/argilla_server/apis/errors/v1/exception_handlers.py

```diff
@@ -19,6 +19,16 @@
 
 
 def add_exception_handlers(app: FastAPI):
+    @app.exception_handler(errors.NotFoundError)
+    async def not_found_error_exception_handler(request, exc):
```

Comment:
I realize these methods are async? Is this a fastAPI requirement?

Reply:
It's how the examples in the FastAPI documentation are implemented: https://fastapi.tiangolo.com/tutorial/handling-errors/?h=#install-custom-exception-handlers